src/command/jc.jl

```diff
@@ -0,0 +1,140 @@
+# TODO: temporary until the start of JuliaCon
+# JuliaCon.debugmode(true)
+
+function commander(c::Client, m::Message, ::Val{:julia_con})
+    startswith(m.content, COMMAND_PREFIX * "con") || return
+
+    regex = Regex(COMMAND_PREFIX * raw"con( help| juliacon2021| now| today)? *(.*)$")
+
+    matches = match(regex, m.content)
+
+    if matches === nothing || matches.captures[1] ∈ (" help", nothing)
+        help_commander(c, m, :julia_con)
+    elseif matches.captures[1] == " juliacon2021"
+        @info "juliacon2021 was required" m.content m.author.username m.author.discriminator
+        _juliacon2021(c, m)
+    elseif matches.captures[1] == " now"
+        @info "now was required" m.content m.author.username m.author.discriminator
+        _now(c, m)
+    elseif matches.captures[1] == " today"
+        @info "today was required" m.content m.author.username m.author.discriminator
+        _today(c, m)
+    else
+        reply(c, m, "Sorry, are you looking for information on JuliaCon 2021? Please check out `con help`")
+    end
+    return nothing
+end
+
+function help_commander(c::Client, m::Message, ::Val{:julia_con})
+    reply(c, m, """
+        How to look for information about JuliaCon 2021 with the `con` command:
+        ```
+        con help
+        con juliacon2021
+        con now
+        con today
+        ```
+        `help` prints this message
+        `juliacon2021` greets the users and provide a link to the official website
+        `now` details the talks and events occuring right now on the different tracks
+        `today` lists the talks and event of the day
+        """)
+end
+
+function _juliacon2021(c::Client, m::Message)
+    reply(c, m, "Welcome to JuliaCon 2021! Find more information on https://juliacon.org/2021/.")
```

Comment:
```suggestion
    reply(c, m, "Welcome to JuliaCon 2021! Find more information at https://juliacon.org/2021/.")
```

Reply:
Haha, this is copy/paste from JuliaCon (and I made a PR to handle this from there, so, maybe, I need to modify it there too :D